modules/programs/hstr.nix

```
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let cfg = config.programs.hstr;
+in {
+  meta.maintainers = [ maintainers.Dines97 ];
```

The CI job fails because you are not present in the maintainers attribute set. You can add yourself in `lib/maintainers.nix` and reference that

```suggestion
  meta.maintainers = [ hm.maintainers.Dines97 ];
```

---

Didn't know that fixing right now

---

Why hm.maintainers instead of just maintainers?

---

Without the `hm` it refers to the [Nixpkgs maintainers list](https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix).